site/src/_pages/docs/0_index.mdx

```diff
@@ -7,10 +7,10 @@ These pages provide a practical introduction to implementing the block protocol,
 
 For the full technical details, read [the Block Protocol specification](https://blockprotocol.org/spec).
 
-If you have any questions, [please open or contribute to a discussion](https://github.com/blockprotocol/blockprotocol/discussions).
+If you have any questions, [please open or contribute to a discussion](https://github.com/blockprotocol/blockprotocol/discussions). You can also [chat to us on Discord](https://blockprotocol.org/discord).
 
-## Demo application
+## HASH: a demo application
```

Comment:
```suggestion
## Demo applications
```

I think the original works better here - we don't want the BP spec to appear 'HASH-branded'.

Reply:
Also suggest plural form rather than singular to imply openness to featuring others.